ynr/apps/api/v09/views.py

```diff
@@ -178,7 +178,10 @@ class CurrentElectionsView(View):
 
     def get(self, request, *args, **kwargs):
         results = {}
-        for election in Election.objects.filter(current=True).order_by("id"):
+        qs = Election.objects.filter(current=True).order_by("id")
+        if request.GET.get("future"):
+            qs = qs.future()
+        for election in qs:
```

Comment:
Can we apply this to the equivalent endpoint in `/next` (also, shouldn't we be dogfooding `/next` ourselves here)

Reply:
This is an odd case: the view is mounted at `api/current-elections` (without a version prefix), but the code lives in `v09` because of the way I moved the code there in the first place. 

Ideally we'd move to a filter on the `ElectionViewSet` (`current|future`) and the JS would consume that. This endpoint is really only used by the JS used to show the dropdown. It's not documented anywhere.

Happy to do that now (it's annoying it's not neater), but that's a bit of a scope creep

Reply:
so currently, this code:

https://github.com/DemocracyClub/yournextrepresentative/blob/be1d0cb384a2a439d61ea58697b12e62e447913a/ynr/apps/api/next/views.py#L176-L193

is unreachable then?

If so, can we either:
* delete the code in `/next` or
* make the same change here, switch the `api/current-elections` route to point at that and delete the code in `v0.9`?

Reply:
👍 Removed from `next` as I want to move to doing this "properly" in future.